plugins/uefi-capsule/efi/meson.build

```diff
@@ -155,16 +155,23 @@ so = custom_target('fwup.so',
                              ['-lefi', '-lgnuefi', libgcc_file_name])
 
 configure_file(
-  input : 'generate_binary.sh.in',
-  output : 'generate_binary.sh',
+  input : 'generate_binary.py.in',
+  output : 'generate_binary.py',
   configuration : conf
 )
 
-build_tool = join_paths(meson.current_build_dir(), 'generate_binary.sh')
+configure_file(
```

Comment:
i think you want:

    sbat_cvs = configure_file(....

and then

    command : [build_tool, '@INPUT@', '@OUTPUT@', efi_format, sbat_cvs],

this will ensure that the sbat file is built before the efi binary; it also means you get an absolute path as the 5th argument, which means you can drop the `"%s/sbat.csv" % build_dir` construct too.

Reply:
Indeed. I'll do that.